Guideline how to contribute to library (#185)
DLR-SR/ThermofluidStream: CONTRIBUTING.md

Thanks for considering to contribute to the DLR Thermofluid Stream Library.

> Contributing can be done by either filing an issue or by contributing code.

Reviewer: ```suggestion
You can contribute either by reporting an issue or by contributing code.
```

> When filing an issue, please include all information necessary to reproduce,

Reviewer: ```suggestion
When filling an issue, please include all information necessary to reproduce,
```

Author: Hier kommt "filing" von "to file - einreichen", das sollte dann mit einem "l" passen
